From f949498897a0916cc8fc0320c37025a2c1eb4d14 Mon Sep 17 00:00:00 2001
From: Jiri Benc <jbenc@redhat.com>
Date: Fri, 12 Sep 2014 14:30:00 -0400
Subject: [net] openvswitch: Fix typo

Message-id: <a98fd0f1f335f7774c136b243561103a3bdf8f29.1410531783.git.jbenc@redhat.com>
Patchwork-id: 94000
O-Subject: [RHEL7.1 net PATCH 27/50] openvswitch: Fix typo.
Bugzilla: 1110384
RH-Acked-by: Flavio Leitner <fbl@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110384
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7948291

commit eb07265904d6ee95497aba0f3cbd2ae6d9c39a97
Author: Jarno Rajahalme <jrajahalme@nicira.com>
Date:   Mon May 5 14:15:18 2014 -0700

    openvswitch: Fix typo.

    Incorrect struct name was confusing, even though otherwise
    inconsequental.

    Signed-off-by: Jarno Rajahalme <jrajahalme@nicira.com>
    Signed-off-by: Pravin B Shelar <pshelar@nicira.com>

Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/openvswitch/flow_table.c b/net/openvswitch/flow_table.c
index c80df6f..574c3ab 100644
--- a/net/openvswitch/flow_table.c
+++ b/net/openvswitch/flow_table.c
@@ -139,7 +139,7 @@ static void flow_free(struct sw_flow *flow)
 {
  int node;
 
- kfree((struct sf_flow_acts __force *)flow->sf_acts);
+ kfree((struct sw_flow_actions __force *)flow->sf_acts);
  for_each_node(node)
   if (flow->stats[node])
    kmem_cache_free(flow_stats_cache,
-- 
1.7.1